actions/python-ql/action.yml

```diff
@@ -87,7 +106,8 @@ runs:
     - name: Integration Testing
       if: inputs.integration-test-path
       run: |
-        if [[ -f "${{ inputs.integration-test-path }}" ]]; then
+        set -e
+        if [[ -n "${{ inputs.integration-test-path }}" && -f "${{ inputs.integration-test-path }}" ]]; then
```

Comment:
Is there a reason to use  `-n && -f` instead of just `-f` as condition? Occurs multiple times in file.